### DISCUSSION TITLE
Prevent scrolling main contents, when navbar is opened

### DISCUSSION THREAD

**Discussion by borachoidev** [mantinedev/mantine#4293] [Feature requests]
Is there any solution to prevent scrolling main contents, when navbar is opened?
main contents is showing through the navbar.

https://github.com/mantinedev/mantine/assets/66766189/cd577375-273a-471f-b450-065d3fa3038d

**Comment by rtivital** [comment]
You can use react-remove-scroll library – https://www.npmjs.com/package/react-remove-scroll

**Comment by ahmeddotgg** [comment]
do you find any solution? 
other than installing another library

**Reply by borachoidev** [reply]
Not yet. I installed react-remove-scroll library as a solution.
